.gitignore

```diff
@@ -146,3 +147,4 @@ dmypy.json
 #custom file everyone has
 Miniconda3*.sh
 Miniconda3*.exe
+user_data/workflows/workflow_Test.yaml
```

Comment:
Ist das hier noch aktuell oder hast du das nur zum ausprobieren verwendet?
Falls ersteres würde  ich hinterfragen, ob das hier tatsächlich hingehört, die anderen gitignore Dinge hier sind irgendwie vom Konzept her anders oder? Kannst du mal erklären, warum du dich dazu entschieden hast? :)

Reply:
Ich war hier auch dabei und das Problem ist, dass ansonsten die Workflows mit commited und gepushed werden. Aber das im .gitignore zu haben ist auch keine Lösung, da stimme ich zu. Hast du eine andere Idee? Das Problem tritt ja auch eigentlich jedes Mal auf :/

Reply:
Ach echt? Aktuell pushen wir immer workflows mit? (Klingt auch nach einer Frage für das alte BP)

Reply:
Ich hatte meine selbst erstellten Workflows erstmal wieder gelöscht und nur auf bestehenden gearbeitet. Dann kann das Problem erst einmal umgangen werden.

Reply:
Lösung gefunden: Wir räumen das mal auf und fügen das dann in die gitignore hinzu